dev_guide/pod_autoscaling.adoc

```diff
@@ -205,7 +205,7 @@ metadata:
   name: hpa-resource-metrics-memory <1>
 spec:
   scaleTargetRef:
-    apiVersion: apps/v1beta1 <2>
+    apiVersion: apps/v2beta1 <2>
```

Comment:
whoops, this shouldn't have changed -- that's a reference to the `apps` API group (but I think it can be `apps/v1` now)